.github/workflows/node.yml

```diff
@@ -75,6 +80,10 @@ jobs:
           npm ci
           npm run build --if-present
 
+      - name: Check webpack build changes
+        run: |
+          bash -c "[[ ! \"`git status --porcelain `\" ]] || (echo 'Please recompile and commit the assets, see the section \"Show changes on failure\" for details' && exit 1)"
+
```

Comment:
but there is no Webpack 👀

Reply:
IMO the name should be fixed in the templates